cosmetics-web/app/models/elasticsearch_query.rb

```diff
@@ -0,0 +1,59 @@
+class ElasticsearchQuery
+  def initialize(keyword, category)
+    @keyword = keyword
+    @category = category
+  end
+
+  def build_query
+    query
+  end
+
+  def query
+    {
+      query: {
+        bool: {
+          must: search_query,
+          filter: filter_query,
+        },
+      },
+    }
+  end
```

Comment:
Please feel free to ignore this if both `build_query` and `query` are needed. 
Seems that the public interface needs `build_query`, while `query` itself is not called anywhere. But I may be missing something.

```suggestion
  def build_query
    {
      query: {
        bool: {
          must: search_query,
          filter: filter_query,
        },
      },
    }
  end
```